internal/pkg/cli/env_deploy.go

```diff
@@ -223,6 +226,15 @@ func (o *deployEnvOpts) Execute() error {
 		}
 	}
 	if err := deployer.DeployEnvironment(deployInput); err != nil {
+		var errStackDeletedOnInterrupt *deploycfn.ErrStackDeletedOnInterrupt
```

Comment:
Can we flip the logic to make less code indented?

```go
err := deployer.DeployEnvironment(deployInput)
if err == nil {
  return nil
}
```
and a switch statement to check the `err` type.